photo-app/config/initializers/stripe.rb

```diff
@@ -0,0 +1,6 @@
+Rails.configuration.stripe = {
+  :publishable_key => ENV['PUBLISHABLE_KEY'],
+  :secret_key      => ENV['SECRET_KEY']
```

Comment:
How did you set these env ?

Reply:
exported from a `.zshrc` file. I added to .gitignore

Reply:
Thats where you set the env.
Now, think. If you are on a team, how would everyone knows what are the new env vars that you have (or add) in your project?

Reply:
🤔No idea, I searched around only found libraries or 3rd party services to share passwords/credentials within the team. What would be your recommendation?